Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original

```diff
@@ -94,6 +96,10 @@ static const int kMSStartupType = 0 /*STARTUP_TYPE*/;
 
 #ifdef APPCENTER_UNITY_USE_DISTRIBUTE
 
+NSNumber *updateTrack = [[NSUserDefaults standardUserDefaults] objectForKey:kMSUpdateTrackKey];
```

Comment:
Why is this read from storage but the editor is just replacing a constant. If the editor is editing code I don't understand why storage is needed here.

Reply:
It was done for the test app. But if we don't need test app changes then it should be deleted.